### PR TITLE
Badge component page

### PR DESCRIPTION
**Pull request by mcmonojit**
**Added Badge component page to documentation**

- Status badge
- Notification badge

**Fixes in avatar and alert**

In order to centre the components flex property was used and consequently the alert and avatar components became disoriented.  To fix this issue the components were wrapped inside a div.

### CODE REVIEW

File: css/components/badge.css [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+    --notification: #581c87;
+}
+
+.badge-container{
+    position: relative;
+    display: inline-block;
+}
+
+.badge{
+    background-color: #365314;
```
Comment (ravikumar0403) [inline css/components/badge.css]: use variable for this color as well

File: docs/avatar/avatar.css [hunk 2]
```diff
@@ -2,9 +2,9 @@
 @import url(/css/allcomponent.css);
 
 .comp-container-width1{
-    max-width: 35rem;
+    /* max-width: 35rem; */
```
Comment (ravikumar0403) [inline docs/avatar/avatar.css]: delete this commented code if not required

File: docs/badge/badge.css [hunk 3]
```diff
@@ -0,0 +1,10 @@
+@import url(../docs.css);
+@import url(/css/allcomponent.css);
+
+.comp-container-width1{
+    /* max-width: 35rem; */
+}
+
+.code-snip-width{
+    /* max-width: 52rem; */
```
Comment (ravikumar0403) [inline docs/badge/badge.css]: delete if not required

**Review by ravikumar0403 — APPROVED** [review]
Looks good.. Left a couple of sanity comments

**Review by chanchal16 — APPROVED** [review]
Remove the commented code and unnecessary blank spaces. Overall looks good!

**Pull request merged**